src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt

```diff
@@ -581,13 +550,10 @@ private fun checkClickEvents(target: EventTarget?, typesToCheck: Collection<Even
             } else {
                 val content = target.dataset[DATA_CLICK_EVENT_VALUE.camel] ?: ""
                 bulmaToast.toast(
-                    json(
-                        "message" to
-                            "<p><b>Click Event</b></p><p>Action: <i>${
-                            clickAction.replace('_', ' ').replaceFirstChar(Char::uppercase)
-                            }</i></p><p>Content: <i>$content</i></p>",
-                        "type" to "is-info"
-                    )
+                    "<p><b>Click Event</b></p><p>Action: <i>${
+                    clickAction.replace('_', ' ').replaceFirstChar(Char::uppercase)
```

Comment:
this is so ugly.. not sure if there's anything we could do here - perhaps a text block? or an extension function or something?